backend/Startup.cs

```diff
@@ -13,91 +13,93 @@
 
 namespace WhaleSpotting
```

Comment:
Please revert all changes in this file! It looks like you ran a formatter on it? You should avoid making any changes that aren't related to your task to avoid merge conflicts, especially formatting changes like this (because git really doesn't like those!)